src/Shared/EndpointMetadataProviderRunner.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http.Metadata;
+using Microsoft.Extensions.Internal;
+
+#nullable enable
+
+namespace Microsoft.AspNetCore.Http;
+
+internal static class EndpointMetadataPopulator
```

Comment:
Is this a dup? https://github.com/dotnet/aspnetcore/blob/1f057843d64b59c9dfe61ce4892ca9afaa664428/src/Shared/EndpointMetadataPopulator.cs

Reply:
Good catch!